### DISCUSSION TITLE
no AssetLoader found for the following extension: ldtk     in bevy_ecs_ldtk

### DISCUSSION THREAD

**Discussion by Abubakar1122331** [bevyengine/bevy#9412] [Q&A] [answered]
The title says it all

I'm using

bevy = "0.11.0"
bevy_ecs_ldtk = "0.8.0"

**Comment by rparrett** [comment, marked as answer]
It sounds like maybe you haven't added the `LdtkPlugin` to your app which would include that asset loader. See the commented line in this excerpt from the ["basic" example](https://github.com/Trouv/bevy_ecs_ldtk/blob/v0.8.0/examples/basic.rs). 
```rust
App::new()
    .add_plugins(
        DefaultPlugins.set(ImagePlugin::default_nearest()),
    )
    .add_plugins(LdtkPlugin) // <---- this line
    .add_systems(Startup, setup)
    .insert_resource(LevelSelection::Index(0))
    .register_ldtk_entity::<MyBundle>("MyEntityIdentifier")
    .run();
 ```


The [bevy_ecs_ldtk repo](https://github.com/Trouv/bevy_ecs_ldtk) or the [bevy_ecs_ldtk help thread](https://discord.com/channels/691052431525675048/1034543904478998539/threads/1037525851362840636) on discord might be a better place to find help with that plugin.
